models/survey.py

```diff
@@ -174,8 +174,8 @@ def get_bar_data(question_data, question):
                 r.branch(
                     (question_data['response_format'] == Question().RESPONSE_MULTIPLE_CHOICE),
                     {
-                        'labels': question[1].distinct(),
-                        'series': [question[1].distinct().map(lambda val: question[1].filter(lambda foo: foo == val).count())]
+                        'labels': question_data['options'],
+                        'series': question[1]
```

Comment:
Alternatively I've been trying this:  
question[1].fold([], function(acc, result) {
                                return acc.add(result);
                            });  
But I have a syntax error :(  
@untra